modules/desktop/profiles/graphics.nix

```diff
@@ -74,12 +74,15 @@ in
   };
 
   config = mkIf cfg.enable {
-    hardware.graphics.enable = true;
+    hardware.graphics = {
+      enable = true;
+      extraPackages = [ pkgs.intel-media-driver ];
+    };
     environment.noXlibs = false;
     environment.sessionVariables = {
       WLR_RENDERER = cfg.renderer;
       XDG_SESSION_TYPE = "wayland";
-      WLR_NO_HARDWARE_CURSORS = 1;
+      WLR_NO_HARDWARE_CURSORS = 0;
```

Comment:
I think this is good. We could also drop the `WLR_NO_HARDWARE_CURSORS=0`as it's the default but this makes it more explicit to bring cursors SW rendering back. It would be good to mention on line 41 that this must be set if SW renderer `pixman` is chosen.

Reply:
Optionally (preferred) - set this value based on the `renderer` value with following pseudo code:
```
if renderer == pixman: WLR_NO_HARDWARE_CURSORS = 1
elif renderer == gles2: WLR_NO_HARDWARE_CURSORS = 0
```